StreamingKit/StreamingKit.xcodeproj/project.pbxproj

```diff
@@ -724,7 +724,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 4.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 5.0;
```

Comment:
@haritowa why do we need to update deployment target?

Reply:
As far as I can see, this project is using ARC, but `__weak` references are not supported prior to 5.0.

If you will try to build this project with the old target - you will encounter 
>cannot create __weak reference because the current deployment target does not support weak references

As [this](https://developer.apple.com/library/content/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html) document stands, 
> Weak references are not supported in OS X v10.6 and iOS 4.

Reply:
Thank you for detailed explanation!